report-builder/src/jetbrains/coverage/report/impl/html/HTMLReportBuilderImpl.java

```diff
@@ -40,13 +40,15 @@ public class HTMLReportBuilderImpl implements HTMLReportBuilder {
   private File myReportZip;
   private String myResourceBundleName = "javaCoverage";
   private String myReportTitle = "";
+  private String myCharset = "";
```

Comment:
I think that we can set `Charset.defaultCharset().displayName()` here by default

Reply:
Good idea. i fixed it!